x-pack/plugins/siem/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -86,57 +85,8 @@ export const getNetworkTopNFlowColumns = (
       const ip: string | null = get(ipAttr, node);
 
       if (Array.isArray(domains) && domains.length > 0) {
-        const domain = domains[0];
-        const id = escapeDataProviderId(
-          `${tableId}-table-${ip}-${flowTarget}-${flowDirection}-domain-${domain}`
-        );
-        return (
-          <DraggableWrapper
-            key={id}
-            dataProvider={{
-              and: [],
-              enabled: true,
-              id,
-              name: domain,
-              excluded: false,
-              kqlQuery: '',
-              queryMatch: { field: domainAttr, value: domain },
-            }}
-            render={(dataProvider, _, snapshot) =>
-              snapshot.isDragging ? (
-                <DragEffects>
-                  <Provider dataProvider={dataProvider} />
-                </DragEffects>
-              ) : (
-                <>
-                  {domain}
-                  {domains.length > 1 && (
-                    <EuiToolTip
-                      content={
-                        <>
-                          {domains.slice(1, 6).map(domainName => (
-                            <span key={`${id}-${domainName}`}>
-                              {defaultToEmptyTag(domainName)}
-                              <br />
-                            </span>
-                          ))}
-                          {domains.slice(1).length > 5 && (
-                            <b>
-                              <br />
-                              {i18n.MORE}
-                            </b>
-                          )}
-                        </>
-                      }
-                    >
-                      <MoreDomains type="eye" />
-                    </EuiToolTip>
-                  )}
-                </>
-              )
-            }
-          />
-        );
+        const id = escapeDataProviderId(`${tableId}-table-${ip}-${flowDirection}`);
+        return getRowItemDraggables(domains, domainAttr, id, undefined, 1);
```

Comment:
I would imagine using named arguments with object destructoring would look cleaner  rather than having to pass in `undefined` here. Optional change, fine if you want to keep it. Just my 2 cents.